dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/entities/MessageEntity.kt

```diff
@@ -0,0 +1,63 @@
+package org.dataland.datalandcommunitymanager.entities
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.dataland.datalandcommunitymanager.model.dataRequest.StoredDataRequestMessageObject
+import java.time.Instant
+import java.util.UUID
+
+/**
+ * The database entity for storing a single message in a data request conversation
+ */
+@Entity
+@Table(name = "messages")
+data class MessageEntity(
+    @Id
+    val messageId: String,
+
+    val ordinal: Int,
```

Comment:
Könnten wir durch einen creationTimestamp ersetzen.  
Das würde auch Ordering ermöglichen, + man könnte in der UI anzeigen wann messages geschrieben wurden.

Sollte dann "lastModifiedDate" nicht gelöscht werden aus dieser Klasse?

Reply:
Genau, das war das was ich als to discuss markieren wollte :D Das klären wir dann also auch am Montag nach dem Daily

Reply:
Wie besprochen, erledigt.